proto/config/edge_node_cluster.proto

```diff
@@ -0,0 +1,37 @@
+// Copyright(c) 2024 Zededa, Inc.
+// All rights reserved.
+
+syntax = "proto3";
+
+import "evecommon/evecommon.proto";
+import "evecommon/acipherinfo.proto";
+
+package org.lfedge.eve.config;
+option go_package = "github.com/lf-edge/eve-api/go/config";
+option java_package = "org.lfedge.eve.config";
+
+message EdgeNodeCluster {
```

Comment:
Will this be part of EdgeDevConfig?

Reply:
yes

Reply:
I think Milan was pointing out that in this PR it is not part of EdgeDevConfig hence it is unused.

Reply:
Ah, got it. I have added:

message EdgeDevConfig {

...

    // cluster configure

    EdgeNodeCluster cluster;

}
in lfedge wike doc.

We need to add here in this PR.

Reply:
Added the field to EdgeDevConfig